fillers/eips/eip3860.py

```diff
@@ -457,6 +457,19 @@ def generate_create_opcode_initcode_test_cases(
     """
     env = Environment()
 
+    call_code = Yul(
+        """
+        {
+            calldatacopy(0, 0, calldatasize())
+            let call_result := call(5000000, \
+                0x0000000000000000000000000000000000000100, \
+                0, 0, calldatasize(), 0, 0)
+            sstore(0, call_result)
+            sstore(call_result, 1)
```

Comment:
```suggestion
            sstore(call_result, 1)
```
Only line 468 should remain.